kubectl-rook-ceph.sh

```diff
@@ -161,7 +161,7 @@ function run_help_command () {
 
 function run_ceph_command () {
   # do not call end_of_command_parsing here because all remaining input is passed directly to 'ceph'
-  kubectl --namespace "$NAMESPACE" exec deploy/rook-ceph-operator -- ceph "$@" --conf=/var/lib/rook/rook-ceph/rook-ceph.config
+  kubectl --namespace "$NAMESPACE" exec deploy/rook-ceph-operator -- ceph "$@" --conf=/var/lib/rook/"$NAMESPACE"/"$NAMESPACE".config
```

Comment:
Should we have a single variable for the path of the ceph conf instead? Like a top value:

`CEPH_CONF_PATH=/var/lib/rook/"$NAMESPACE"/"$NAMESPACE".config`

So future reference can be changed in one place?

Reply:
make sense, changed